store/streaming/constructor.go

```diff
@@ -67,7 +67,10 @@ func NewServiceConstructor(name string) (ServiceConstructor, error) {
 func NewFileStreamingService(opts serverTypes.AppOptions, keys []types.StoreKey, marshaller codec.BinaryCodec) (baseapp.StreamingService, error) {
 	filePrefix := cast.ToString(opts.Get("streamers.file.prefix"))
 	fileDir := cast.ToString(opts.Get("streamers.file.write_dir"))
-	return file.NewStreamingService(fileDir, filePrefix, keys, marshaller)
+	// NOTICE: the default configuration don't guarantee data integrity of output to gain higher performance,
+	// if you need data integrity, pass `true` to the parameters `stopNodeOnErr` and `fsync`, be aware of the performance penalty of that.
+	// TODO the configuration system will be improved later with the plugin system.
```

Comment:
Can you add a `// Ref: ...` comment to indicate the relevant issue and/or PR please?

Reply:
done